### PR TITLE
Fix/common/net/unstable

### PR DESCRIPTION
**Pull request by mightytry**

### CODE REVIEW

File: desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt [hunk 1]
```diff
@@ -146,8 +146,8 @@ class MovementHandlerViewModel {
     fun endDrag(pictureSize: Size) {
         if (!dragPoint.value.toTopLeftOrigin(pictureSize).isInBounds(pictureSize.toRect())
         ) return
-        println(dragPoint.value.toTopLeftOrigin(pictureSize).normalize(pictureSize))
         addClick(dragPoint.value.toTopLeftOrigin(pictureSize).normalize(pictureSize))
+        println(dragPoint.value.toTopLeftOrigin(pictureSize).normalize(pictureSize))
```
Comment (GrumpyDucc) [inline desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt:150]: remove this line

**Review by RobertFrank06 — APPROVED** [review]
image

File: desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt (hunk 1 above)
Comment (GrumpyDucc) [inline desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt:150]: remove this line

**Review by GrumpyDucc — CHANGES_REQUESTED** [review]

**Comment by GitbertFrank** [comment]
kys nigger

**Comment by GitbertFrank** [comment]
image

File: desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt (hunk 1 above)
Comment (GrumpyDucc) [inline desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt]: pls remove this debug statement

**Review by GrumpyDucc — CHANGES_REQUESTED** [review]

File: desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt (hunk 1 above)
Comment (RobertFrank06) [reply desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt]: Removed it
image

**Pull request merged**